code/go/0chain.net/miner/miner/main.go

```diff
@@ -204,7 +204,28 @@ func main() {
 	initIntegrationsTests(node.Self.Underlying().GetKey())
 
 	var server *http.Server
+	var profServer *http.Server
+
+	mux := http.NewServeMux()
+	http.DefaultServeMux = mux
+
 	if config.Development() {
+		if viper.GetBool("development.pprof") {
+			// start pprof server
+			pprofMux := http.NewServeMux()
+			profServer = &http.Server{
+				Addr:           fmt.Sprintf(":%d", node.Self.Underlying().Port+10),
```

Comment:
Do not use this `+10` port, which will have ports confliction probelm when we have more than 10 miners. Define it in config file and use a specific port number, e.g 6060, etc.

Reply:
Valid point, I will change the range. But this is for development only. And if you look at out docker-compose file for local deployment, it exposes ports on 717(*), which means exactly 9 ports. Thus a clash seems unlikely.